src/model/mod.rs

```diff
@@ -27,6 +35,25 @@ pub struct Manifest {
     pub spec: Specification,
 }
 
+impl Manifest {
+    /// Returns a reference to the current version
+    pub fn version(&self) -> &str {
+        self.metadata
+            .annotations
+            .get(VERSION_ANNOTATION_KEY)
+            .map(|v| v.as_str())
+            .unwrap_or_default()
```

Comment:
Is an empty string okay here? Seems like this might need to generate an error

Reply:
According to the OAM spec, it is perfectly valid to _not_ have a version. However, in wadm you could only have one version with `""`

Reply:
Did we want to opinionated pick something like `v0.0.1` like we did in wasmCloud?

Reply:
I'd rather be spec compliant here than pick something by default (especially because we don't enforce semver on it)